CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 All notable changes to this project will be documented in this file.
 
+## [2.0.0] - 2020-11-11
+
+### Added
+- Support for DB export by routing export requests to the corresponding BulkDataAccess interfaces as defined in `fhir-works-on-aws-interface` v3.0.0
+- Supporting capability statement configuration for OAuth as defined in `fhir-works-on-aws-interface` v3.0.0
+- Improved error handling to allow matching of same error objects across different `fhir-works-on-aws-interface` versions 
+

```

Comment:
Don't forget to mention the CORS support on the changelog. It's not part of this merge but it is part of the 2.0.0 release.

Btw, I know that we haven't standarized our release process, but I don't like modifying the changelog on the same commit that is adding the feature. I think that is cleaner to do a separate commit specifically for release stuff (changelog, version bump, etc). That creates a commit history that is easier to understand.

Reply:
updated